api/service/stagedstreamsync/metric.go

```diff
@@ -82,6 +94,18 @@ var (
 		[]string{"ShardID"},
 	)
 
+	numBlocksInsertedEpochSyncHistogramVec = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: "hmy",
+			Subsystem: "staged_stream_sync",
+			Name:      "num_blocks_inserted_epoch_sync",
+			Help:      "number of blocks inserted for each epoch sync",
```

Comment:
what do you mean here by  `each epoch sync`

Reply:
the number of blocks have been inserted by epoch sync

Reply:
it's the word "each" that bothers me. Does it mean we have multiple epoch sync ? if yes wouldn't have number always be 1 ? as it would increment at every epoch ?

Reply:
Actually it is a cycle of **epoch synchronization**, where each cycle attempts to add any blocks that were missed on the **epoch chain**. The number of missed blocks can vary; it might be just one or multiple blocks, depending on factors such as node crashes or the node being stuck for a period of time.

Reply:
BTW, any suggestion for name? I can change it to whatever makes more sense